pkg/virt-operator/kubevirt.go

```diff
@@ -44,6 +44,10 @@ import (
 	operatorutil "kubevirt.io/kubevirt/pkg/virt-operator/util"
 )
 
+const (
+	virtOperatorJobAppLabel = "virt-operator"
```

Comment:
Can we use a specific label (`virt-operator-strategy-dumper` for instance)? Otherwise we have the risk that other layered entities (services, prometheus collectors, ...) may by accident think that this is a virt-operator instance.

Reply:
Sure. My thinking was that is is the same binary and may need the same permissions to potentially do anything virt-operator does (and also a bit of a shortcut to have one policy apply to both), but since it serves a different purpose it makes sense to split them out.

Reply:
using the same virt-operator service account for this job was a mistake. This job only needs the ability to Create/Update a config map. I'm tracking that as a bug here, #3356